### PR TITLE
rv

### PR DESCRIPTION
**Pull request by rvnarvaiza**

### CODE REVIEW

**Comment by rvnarvaiza** [comment]
Leaves Table

File: src/http-request/leave.request.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+export const getLeaveCredits = () => {
+  return new Promise((resolve) => {
+    setTimeout(() => {
+      resolve(leaveData)
+    }, 1000)
+  })
+}
+
+export const getLeaveById: (id: number) => Promise<EmployeeLeave> = (id) => {
```
Comment (allyncubero) [inline src/http-request/leave.request.ts]: - [x] Change Function Name to getLeaveCreditsByEmployeeId

File: src/pages/employee/[employee_id].tsx [hunk 2]
```diff
@@ -3,17 +3,35 @@ import { NextPage } from 'next'
...
+      key: 'leaveType',
+    },
+  ]
 
   useEffect(() => {
     const employee_id = router.query.employee_id
     const id = parseInt(employee_id as string)
     getEmployeeById(id).then((e: EmployeeModel) => setData(e))
+    console.log(data)
+    getLeaveById(id).then((e: EmployeeLeave) => setLeaves(e))
```
Comment (allyncubero) [inline src/pages/employee/[employee_id].tsx]: - [ ] on the get employee by id then callback, call getLeaveCreditsByEmployeeId, 
- [ ] remove console log

**Review by allyncubero — CHANGES_REQUESTED** [review]

File: src/http-request/leave.request.ts (hunk 1 above)
Comment (rvnarvaiza) [reply src/http-request/leave.request.ts]: done na po...please check po, pedalking

File: src/pages/employee/[employee_id].tsx (hunk 2 above)
Comment (rvnarvaiza) [reply src/pages/employee/[employee_id].tsx]: done na po....please check po, deathvalley ironman..

**Review by allyncubero — APPROVED** [review]
LGTM

**Pull request merged**